src/DotNet.Testcontainers/Containers/Builders/TestcontainersBuilder.cs

```diff
@@ -196,6 +197,7 @@ public TDockerContainer Build()
 
     private static ITestcontainersConfiguration Apply(
       Uri endpoint = null,
+      string gateway = null,
```

Comment:
Do we really need an additional parameter `gateway`? Couldn't we just use `endpoint`? I think that's much simpler.